gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -724,6 +906,169 @@ def call_metric(self, other_args: List[str]):
+        """Process cf command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="cf",
+            description="See all cash flow statement metrics available",
+        )
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if ns_parser:
+            help_text = """
+        ninc         net income
+        da           depreciation & amortization
+        sbc          share-based compensation
+        ooa          other operating activities
+        ocf          operating cashflow (ni + da + sbc + ooa)
+        cex          cash expenditures
+        acq          acquisitions
+        cii          change in investments
+        oia          other investing activities
+        icf          investing cashflow (cex + acq + cii + oia)
+        dp           dividends paid
+        si           share issuance / repurchase
+        di           debt issued / paid
+        ofa          other financing activities
+        fcf          financing cashflow (dp + si + di + ofa)
+        ncf          net cashflow (ocf + icf + fcf)
+            """
+            console.print(help_text)
+
+    def call_visualise(self, other_args: List[str]):
+        """Process visualise command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="visualise",
+            description="Visualize a particular metric with the filters selected",
+        )
+        parser.add_argument(
+            "-m",
+            "--metric",
+            dest="metric",
+            required="-h" not in other_args,
+            help="Metric to visualize",
+            choices=self.visualise_choices,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            default=10,
+            help="Limit number of companies to display",
+            type=check_positive,
+        )
+
+        parser.add_argument(
+            "-p",
+            "--period",
+            dest="period",
+            default=12,
+            help="Limit number of periods to display",
+            type=check_positive,
+        )
+        parser.add_argument(
+            "-r",
+            "--raw",
+            action="store_true",
+            dest="raw",
+            default=False,
+            help="Output all raw data",
+        )
+
+        if other_args and "-" not in other_args[0][0]:
+            other_args.insert(0, "-m")
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_BOTH_RAW_DATA_AND_FIGURES
+        )
+        if ns_parser:
+            (
+                self.stocks_data,
+                self.tickers,
+            ) = stockanalysis_view.display_plots_financials(
+                ns_parser.metric,
+                self.sa_keys,
```

Comment:
my suggestion is to have these keys on the `_model` file if they are needed there. Then you can just access them through `stockanalysis_model.sa_keys` in the controller.